INSTALL.md

```diff
@@ -0,0 +1,202 @@
+   which guides the user through the installation procedure and suggests
+   sensible defaults for various configure/installation options.
+   We highly recommend using this script to perform the installation.
+
+**2. Installation using the standard GNU build process**
+
+   You may, of course, also use GNU's standard
+
+      configure && make && make install && make check
+
+   procedure, described in more detail below.
+
+## Basic Installation
+
+   These are generic installation instructions for GNU packages.
+
+   The `configure` shell script attempts to guess correct values for
+various system-dependent variables used during compilation.  It uses
+those values to create a `Makefile` in each directory of the package.
+It may also create one or more `.h` files containing system-dependent
+definitions.  Finally, it creates a shell script `config.status` that
+you can run in the future to recreate the current configuration, a file
+`config.cache` that saves the results of its tests to speed up
+reconfiguring, and a file `config.log` containing compiler output
+(useful mainly for debugging `configure`).
+
+   If you need to do unusual things to compile the package, please try
+to figure out how `configure` could check whether to do them, and mail
+diffs or instructions to the address given in the `README.md` so they can
+be considered for the next release.  If at some point `config.cache`
+contains results you don't want to keep, you may remove or edit it.
+
+   The file `configure.in` is used to create `configure` by a program
+called `autoconf`.  You only need `configure.in` if you want to change
+it or regenerate `configure` using a newer version of `autoconf`.
+
+The simplest way to compile this package is:
+
+  1. `cd` to the directory containing the package's source code and type
+     `./configure` to configure the package for your system.  If you're
+     using `csh` on an old version of System V, you might need to type
+     `sh ./configure` instead to prevent `csh` from trying to execute
+     `configure` itself.
+
+     Running `configure` takes awhile.  While running, it prints some
+     messages telling which features it is checking for.
+
+  2. Type `make` to compile the package.
+
+  3. Type `make install` to install the programs and any data files and
+     documentation.
+
+  4. Optionally, type `make check` to run any self-tests that come with
+     the package.
+
+  5. You can remove the program binaries and object files from the
+     source code directory by typing `make clean`.  To also remove the
+     files that `configure` created (so you can compile the package for
+     a different kind of computer), type `make distclean`.  There is
+     also a `make maintainer-clean` target, but that is intended mainly
+     for the package's developers.  If you use it, you may have to get
+     all sorts of other programs in order to regenerate files that came
+     with the distribution.
+
+Compilers and Options
```

Comment:
Replace ===== with ##, we should probably be consistent, and this should probably be a h2 or h3 (===== means h1). There's a few more of these.

Reply:
*facepalm* 
Nice catch! I completely missed that whilst changing the rest of the text. I'll take care of it now. I also think `##` works best for this case so I'll stick to that (unless, of course, you'd prefer I go for h3).